### ISSUE TITLE
hands free not working on Raspberry Pi 3

### ISSUE THREAD

**Issue by faisalch** [Sensory/alexa-rpi#4]
I have Alexa working on my Raspberry Pi 3
have a USB mic and speaker connected via the 3.5mm jack

tried running 'wake word engine' but no luck 

the Java Alexa Voice Service was launched first with 
$ mvn exec:exec

here I can click listen and interact with Alexa 

Then I've run 
cd ~/Desktop/alexa-avs-sample-app/samples/wakeWordAgent/src && ./wakeWordAgent -e sensory

didn't initially load and so i did the following  

working by installing PulseAudio.
sudo apt-get install VLC
sudo apt-get install pulseaudio
sudo apt-get install pavucontrol
The running it with: pulseaudio -D
Then starting the Alexa services.
cd ~/Desktop/alexa-avs-sample-app/samples/wakeWordAgent/src && ./wakeWordAgent -e sensory

then seemed to load ok
Msg I get is - starting Wake Word Engine 

but i still have to press 'Listen' manually 

Any ideas ?

**Comment by Xperia0** [comment]
I have the Same problem.
You had success meanwhile?

**Comment by JHdeVilliers** [comment]
Please verify that the recording and playback works - see item 3 in the [FAQ](https://github.com/Sensory/alexa-rpi#frequently-asked-questions).

**Comment by Xperia0** [comment]
Jeah,  with the Java Button all works.
Only with the wakeword is the problem.

**Comment by iles05** [comment]
yes i confirm  recording and playback works but engine not working....

**Comment by iles05** [comment]
problem is engine capture low rate low volume level voice that's why its not working..

**Comment by iles05** [comment]
i have set 100% vol in both capture & playback device in alsamixer but still same volume level is very low...

**Comment by lennypenny9999** [comment]
I have the identical problem.  When I run the code in the 3rd Window I get error indications.

When I click on the Java Button, the Raspi hears me from across the room.  However, it does not respond to the wake-word

I paste it below:

pi@raspberrypi:~/Desktop/alexa-avs-sample-app/samples $ cd wakeWordAgent/src && ./wakeWordAgent -e sensory
INFO:main: Starting Wake Word Agent
INFO:WakeWordAgent: State set to IDLE(2)
INFO:Initializing Sensory library | library name: TrulyHandsfree | library version: 5.0.0-beta.10.2 | model file: ../ext/resources/spot-alexa-rpi.snsr
WARNING:Library expires on: License expires on 28 Aug 2018 00:00:00 GMT
INFO:SensoryWakeWordEngine: mainLoop thread started
ERROR:An error happened in the mainLoop of SensoryWakeWord snsrRun(): task: Input channel audio-pcm: ALSA error: No such file or directory
INFO:SensoryWakeWordEngine: mainLoop thread ended
INFO:WakeWordAgent: thread started
INFO:WakeWordIPCSocket::mainLoop thread started
INFO:WakeWordIPCSocket: init socket on port:5123
INFO:===> Connected to AVS client <===
